docs/docs/installation/configuring-superset.mdx

```diff
@@ -278,6 +292,29 @@ To use LDAP you must install the [python-ldap](https://www.python-ldap.org/en/la
 See [FAB's LDAP documentation](https://flask-appbuilder.readthedocs.io/en/latest/security.html#authentication-ldap)
 for details.
 
+### Mapping LDAP or OAUTH groups to Superset roles
+
+AUTH_ROLES_MAPPING in Flask-AppBuilder is a dictionary that maps from LDAP/OAUTH group names to FAB roles.
+It is used to assign roles to users who authenticate using LDAP or OAuth.
+
+For example, the following AUTH_ROLES_MAPPING dictionary would map the OAUTH group "superset_users" to the Superset roles "Gamma" as well as "Alpha", and the OAUTH group "superset_admins" to the Superset role "Admin".
```

Comment:
Do you want to add subheaders for OAuth and LDAP to these two configs, to make it clear which is for which - if that's the case?

Reply:
Done